.circleci/config.yml

```diff
@@ -4,12 +4,14 @@ workflows:
   version: 2
   build_and_test_all:
     jobs:
-      - build-test-linux
+      - build-test-linux:
+          context:
+            - hello-world-demos
 
```

Comment:
-^ for access to the SDK key and flag name, so we can actually test a real flag evaluation.